### PR TITLE
media-video/obs-studio: prefer imagemagic-7 and bump to 20.1.3

### PR DESCRIPTION
**Pull request by lucianposton**
- Now that imagemagick-7 is in amd64, adjusted DEPEND so that obs-studio won't hold imagemagick back from upgrading to 7 (without the user knowing to explicitly emerge >=cmake-3.9, which is in ~amd64).
- Added ebuild for 20.1.3

@Chiitoo

### CODE REVIEW

**Comment by lucianposton** [comment]
added https://bugs.gentoo.org/641608 tag to commit body

**Comment by a17r** [comment]
Suggestion: Instead of adding a conditional cmake version dep, set `CMAKE_MIN_VERSION:=3.9.6` above inherit and then open a bug for cmake-3.9.6 stabilisation.

File: media-video/obs-studio/obs-studio-9999.ebuild [hunk 1]
```diff
@@ -50,15 +50,7 @@ COMMON_DEPEND="
...
-	imagemagick? (
-		|| (
-			<media-gfx/imagemagick-7
-			(
-				>=media-gfx/imagemagick-7
-				>=dev-util/cmake-3.9
-			)
-		)
-	)
+	imagemagick? ( >=dev-util/cmake-3.9 )
```
Comment (kuzetsa) [inline media-video/obs-studio/obs-studio-9999.ebuild]: In addition to dev-util/cmake, USE=imagemagick should pull in:

```
>=media-gfx/imagemagick-7
```

(*dropping pre-v7 support, the ebuild will still need to depend on
imagemagick if building with the imagemagick feature enabled*)

**Comment by Chiitoo** [comment]
It might be a bit too soon in my opinion to get rid of the check, since some people may still be using <imagemagick-7 for one reason or another.

**Comment by lucianposton** [comment]
> Suggestion: Instead of adding a conditional cmake version dep, set CMAKE_MIN_VERSION:=3.9.6 above inherit and then open a bug for cmake-3.9.6 stabilisation.

cmake 3.9.6 is stable now, so went ahead and did this

**Comment by Chiitoo** [comment]
Apologies if this was waiting for me.

I believe I'm okay with it, if someone goes ahead and merges this.  ^^

**Pull request closed without merging**